cypher-shell/src/main/java/org/neo4j/shell/cli/CliArgHelper.java

```diff
@@ -161,10 +161,15 @@ private static ArgumentParser setupParser(ParameterMap parameterMap)
                 .setDefault("")
                 .help("password to connect with. Can also be specified using environment variable " + ConnectionConfig.PASSWORD_ENV_VAR);
         connGroup.addArgument("--encryption")
-                .help("whether the connection to Neo4j should be encrypted; must be consistent with Neo4j's " +
-                        "configuration")
-                .type(new BooleanArgumentType())
-                .setDefault(false);
+                .help("whether the connection to Neo4j should be encrypted. This must be consistent with Neo4j's " +
+                      "configuration. If choosing " + Encryption.DEFAULT.name().toLowerCase() +
```

Comment:
wrapping this `Encryption.DEFAULT.name().toLowerCase()` in `"'"` on each side would read more nicely.